tests/test_precompiled.py

```diff
@@ -91,6 +92,28 @@ def test_precompiled_validation_with_preview_calls_overlay_if_pdf_out_of_bounds(
     assert json_data['pages'] == ['SSdtIGEgcG5n']
 
 
+def test_precompiled_validation_with_preview_calls_overlay_if_content_in_new_address_margin(
```

Comment:
It sounds like this tests asserts that the mock was called, but I don't think it actually does at the moment